tests/integrationtest/t/ddl/db_partition.test

```diff
@@ -2270,11 +2271,12 @@ create temporary table nt34 (id int);
 -- error 1733
 alter table pt34 exchange partition p0 with table nt34;
 
---echo ## global index
-create table pt35 (a int, b int, unique index(b)) partition by hash(a) partitions 1;
-create table nt35 (a int, b int, unique index(b));
--- error 1731
-alter table pt35 exchange partition p0 with table nt35;
+# TODO: support EXCHANGE PARTITION with GLOBAL INDEX
```

Comment:
Why comment it? What's the exchange partition behaviour after this PR?

Reply:
Thanks for the suggestion, I reverted the change, added GLOBAL ColumnOption and corrected the error code.